lib/job/arActorExportJob.class.php

```diff
@@ -71,8 +71,33 @@ protected function doExport($path)
                 return;
             }
 
-            $this->exportResource($resource, $path);
+            $this->csvActorExport($path, $resource);
             $this->logExportProgress();
         }
     }
+
+    protected function csvActorExport($path, $resource)
```

Comment:
I would suggest changing the name of this function to avoid confusion with the `csvActionExport` class